documentation/src/main/resources/pages/ditto/connectivity-mapping.md

```diff
@@ -350,6 +351,87 @@ Example configuration:
   }
 }
 ```
+### CloudEvents Mapper
+
+This mapper maps incoming [CloudEvent](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md) to Ditto Protocol. It provides support for both Binary CloudEvents as well as Structured CloudEvents.
+
+**Note**: The mapper only supports incoming messages with `content-type:application/cloudevents+json`. Messages that do not conform to this are ignored.
+
+#### Specifications
+
+Incoming messages need to have the mandatory CloudEvents fields.
+For example, a Binary CloudEvent for Ditto would look like this:
+
+```
+  headers:
+      ce-specversion:1.0
+      ce-id:some-id
+      ce-type:some-type
+      ce-source:generic-producer
+      content-type:application/cloudevents+json
```

Comment:
I am unsure if this is the correct content-type for "binary CloudEvents" ..
Because the payload is cleary DittoProtocol JSON, so content-type should be `"application/vnd.eclipse.ditto+json"`.

But then the CloudEventsMapper would not process the message, or would it.
What do you think @ctron ?

Shouldn't `content-type:application/vnd.eclipse.ditto+json` together with all required "binary CloudEvent" headers also be handled by the CloudEventsMapper?
Currently it isn't ..

Reply:
Right, in this case the content type is the content type of the data which is equal to the payload in the case of a binary encoded event. So it should be `application/vnd.eclipse.ditto+json`.

From what I see, the takes the "ditto headers", which to my understanding are the incoming headers, and passes them along for further processing.